src/app/visual/colorpicker/colorpicker-visual.component.html

```diff
@@ -2,26 +2,39 @@
   (themeSettingsChange)="themeSettingsChange($event)"
 ></sky-docs-demo-control-panel-theme>
 
-<div
-  id="screenshot-colorpicker"
+<button
+  class="sky-btn sky-btn-default"
+  id="screenshot-action-button-center-align"
+  type="button"
+  (click)="onToggleAbleColorpicker()"
 >
-  <sky-colorpicker
-    labelledBy="FOOBAR"
-    #colorPickerExample1
+  Toggle Colorpicker #1's Ability
```

Comment:
Just to be more clear, could we reword this to something like, `Toggle colorpicker disable state` ?

Reply:
done in last commit